3-create_ros_2_package/README.md

```diff
@@ -0,0 +1,111 @@
+# Exercises 3 - Create ROS 2 package
+For the upcoming exercises, you will start writing some Python code.
+ROS 2 organizes code neatly in **packages** to separate different 
+functionalities, and make them to be easily installable and shareable.
+
+You can follow this tutorial to easily create a new package using 
+[Turtle Nest](https://github.com/Jannkar/turtle_nest) GUI tool, or learn how to create packages manually from the command line by following 
+[the official ROS 2 documentation](https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-Your-First-ROS2-Package.html).
+
+## Create a new package
+1. Bring up the Docker container and open a new terminal inside it ([How to run Docker container](/0-setup/Docker%20Cheat%20Sheet.md))
+1. Run a command `turtle-nest` to bring up the guided package creation
+
+Use the following settings to create the package:
+
+**Package name:** `ros2_exercises`
+
+**Destination:** `/home/user/exercises_ws/src`
+
+**Package Type:** `Python`. Uncheck the C++ option
+
+**Python Node Name:** `odometry_publisher` (Will be used for the next exercises)
+
+**Maintainer Name:** `<Your name>`
+
+**Package License:** `No license`
+
+
+Other fields can be left empty / unchecked, as they are not needed in these exercises.
```

Comment:
This is most likely because this setup step was skipped:
`mkdir -p $HOME/exercises_ws/src`

Resolving for now, please open again if the issue persists.